tests/testthat/test-release.R

```diff
@@ -94,6 +94,11 @@ test_that("returns empty string if no bullets", {
   expect_equal(news_latest(lines), "")
 })
 
+test_that("can find milestone numbers", {
```

Comment:
So you seem to be relying on usethis itself being the active project here (?).

We have no other tests that do anything like this.

Given that `gh_milestone_number()` actually hits the GitHub API, we either have to not test this (my recommendation) or refactor it so that one can pass the repo spec in (but then it would still need some skips).

Reply:
This was non-trivial to write, so I wanted to include at least one test. If it causes problems in the future, I promise to fix it.